web_src/css/features/projects.css

```diff
@@ -70,15 +70,20 @@
 .card-attachment-images {
   display: inline-block;
   white-space: nowrap;
-  overflow: hidden;
+  overflow: scroll;
+  cursor: default;
```

Comment:
Why do we need this line?

Reply:
On cards the cursor is a grabbing hand to clue about dragging them around. This reset the cursor hover the image area to help with the scrolling part, though it is still a draggable area. I could have used `cursor: unset`. We can remove it's not necessary.